config/config_helpers.go

```diff
@@ -32,22 +33,52 @@ func ResolveTerragruntConfigString(terragruntConfigString string, include *Inclu
 // Resolve a single call to an interpolation function of the format ${some_function()} in a .terragrunt file
 func resolveTerragruntInterpolation(str string, include *IncludeConfig, terragruntOptions *options.TerragruntOptions) (string, error) {
 	matches := HELPER_FUNCTION_SYNTAX_REGEX.FindStringSubmatch(str)
-	if len(matches) == 2 {
-		return executeTerragruntHelperFunction(matches[1], include, terragruntOptions)
+	if len(matches) == 3 {
+		return executeTerragruntHelperFunction(matches[1], matches[2], include, terragruntOptions)
 	} else {
 		return "", errors.WithStackTrace(InvalidInterpolationSyntax(str))
 	}
 }
 
 // Execute a single Terragrunt helper function and return its value as a string
-func executeTerragruntHelperFunction(functionName string, include *IncludeConfig, terragruntOptions *options.TerragruntOptions) (string, error) {
+func executeTerragruntHelperFunction(functionName string, parameters string, include *IncludeConfig, terragruntOptions *options.TerragruntOptions) (string, error) {
 	switch functionName {
 	case "find_in_parent_folders": return findInParentFolders(terragruntOptions)
 	case "path_relative_to_include": return pathRelativeToInclude(include, terragruntOptions)
+	case "get_env": return getEnvironmentVariables(parameters, terragruntOptions)
 	default: return "", errors.WithStackTrace(UnknownHelperFunction(functionName))
 	}
 }
 
+func extractParametersFromFunction(parameters string) (map[string]string, error) {
+	matches := HELPER_FUNCTION_GET_ENV_PARAMETERS_SYNTAX_REGEX.FindStringSubmatch(parameters)
+	if (len(matches)) < 2 {
+		return nil, errors.WithStackTrace(InvalidFunctionParameters(parameters))
+	}
+	result := make(map[string]string)
+	for value, name := range HELPER_FUNCTION_GET_ENV_PARAMETERS_SYNTAX_REGEX.SubexpNames() {
```

Comment:
I think `index` is a better name than `value`, since that parameter is the index of the iteration.